theano/gpuarray/dnn.py

```diff
@@ -282,10 +282,12 @@ def c_lib_dirs(self):
     def do_constant_folding(self, node):
         return False
 
-    def __init__(self, border_mode, subsample=(1, 1), conv_mode='conv',
-                 precision="float32"):
+    def __init__(self):
         COp.__init__(self, ["conv_desc.c"], "APPLY_SPECIFIC(conv_desc)")
 
+    def make_node(self, border_mode, kern_shape, precision="float32", subsample=(1, 1, 0), conv_mode='conv'):
```

Comment:
keep the same order of parameter as it was. subsample, conv_mode, then precision.